### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by bhtibrewal**

### CODE REVIEW

**Review by Ankur9669 — APPROVED** [review]
Most of the things were already reviewed so cannot find anything in the code, although the auth functionality is not working in deploy preview, so you can check that

**Review by sri-rishi — APPROVED** [review]
code looks good, as Ankur mentioned auth is not working in preview , you should fix that

File: src/services/auth/signUp.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+export const signUp = async ({ data, showToast, navigate }) => {
+    try {
+        const res = await axios.post("/api/auth/signup", data);
+        if (res.status === 201) {
+            navigate("/sign-in");
+            showToast({ title: 'sign up successful   ', type: 'success' });
+        }
+    }
+    catch (e) {
+        console.log(e.response.data.errors);
```
Comment (pankaj-prog) [inline src/services/auth/signUp.js:12]: you can show error toast here, as you are already using toast

**Review by pankaj-prog — APPROVED** [review]
The code is well structured, and the overall work is good.

File: src/services/auth/signUp.js (hunk 1 above)
Comment (bhtibrewal) [reply src/services/auth/signUp.js:12]: yes forgot that, thanks for reminding

**Comment by bhtibrewal** [comment]
> Most of the things were already reviewed so cannot find anything in the code, although the auth functionality is not working in deploy preview, so you can check that

yes the credentials were wrong fixed it

**Pull request merged**